app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,81 @@
+class RecipesController < ApplicationController
+  # GET /recipes or /recipes.json
+  def index
+    @recipes = Recipe.order(:id).page(params[:page]).per(2)
+  end
```

Comment:
The method `page` throws an error. And I can't make sense of what it should do. Can you explain it to me, so I can have more clarity?

Here is a screenshot of the error 
![image](https://user-images.githubusercontent.com/55507343/207398795-cd55b626-9f19-40c8-b24b-97fdff725fc1.png)

Reply:
I used this for pagination but I did not install "kaminari dem" library. For now, I will remove but We will implement it later. Thanks!!!